bitnami/redis/templates/master/psp.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.podSecurityPolicy.create }}
+{{ template "podSecurityPolicy.checkCreate" . }}
```

Comment:
You don't need to add it here since the `"redis.validateValues"` macro is already making this check.
```suggestion
```